apis/v1alpha2/tcproute_types.go

```diff
@@ -92,12 +92,17 @@ type TCPRouteRule struct {
 	// +kubebuilder:validation:MaxItems=8
 	Matches []TCPRouteMatch `json:"matches,omitempty"`
 
-	// ForwardTo defines the backend(s) where matching requests should
-	// be sent.
+	// BackendRefs defines the backend(s) where matching requests should be
+	// sent. If unspecified, this rule MUST not match.
```

Comment:
This seems confusing to me. Why not just require a BackendRef for TCPRoute since it's currently useless without one?

Reply:
I updated this to match HTTPRoute.
One use case: As a user, I want a placeholder TLS/UDP/TCP Route but I don't want to define my Service yet.

Reply:
That just seems confusing to me since there's so little that's actually in these L4 route types. It seems like a placeholder L4 route would contain nothing at all, except maybe a reference to a Gateway, but that would be strange if you didn't already have backends. I'd lean towards keeping the validation strict here with the idea that we can always expand on it in the future.

Reply:
Okay. Let's punt this to later. Updated.